### PR TITLE
writing tip(s)

### PR DESCRIPTION
**Pull request by aivyr**
wrote a writing tip! hopefully I'll be able to add some more to this pull request in future.

### CODE REVIEW

**Comment by kaido1224** [comment]
I just said yesterday that someone should write a writing tip of the day about getting over writer's block.

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4299,6 +4299,28 @@ label monika_write:
     m "Thanks for listening~"
     return
 
+init 5 python:
+      addEvent(Event(persistent.event_database,eventlabel="monika_writingtip4",category=['writing tips'],prompt="Writing tip #4",pool=True))
+  
+label monika_writingtip4:
+     m 3b "Here's Monika's Writing Tip of the Day!"
+     m "You know about writer's block, right?"
+     m 1e "I had it a lot, when I started writing."
```
Comment (MJS103) [inline Monika After Story/game/script-topics.rpy]: m 1e "I had it a lot, when I first started writing."

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4299,6 +4299,28 @@ label monika_write:
...
+     m "Sometimes it was halfway through a draft, but more often before I even started."
```
Comment (MJS103) [inline Monika After Story/game/script-topics.rpy]: m 1l "Sometimes it was halfway through a draft, but more often before I even started."

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -4299,6 +4299,28 @@ label monika_write:
...
+     m 2f "Every time I tried to write a word, I thought, 'this isn't going to sound good', or 'this isn't how I want it to look'. So I'd stop, backtrack, and try again."
+     m 2d "But I realised it ultimately didn't matter if things didn't work out first time!"
+     m 1a "I feel the heart of writing is not about getting it right first time, but about perfecting it afterwards."
```
Comment (MJS103) [inline Monika After Story/game/script-topics.rpy]: m 2a "I feel the heart of writing is not about getting it right first time, but about perfecting it afterwards."

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -4299,6 +4299,28 @@ label monika_write:
...
+     m "The end product is what matters, not the prototype."
```
Comment (MJS103) [inline Monika After Story/game/script-topics.rpy]: m 2a "The end product is what matters, not the prototype."

**Comment by MJS103** [comment]
I really liked this tip, I just had some recommendations on the facial emotions and changed a few words to make the sentence flow slightly better. Otherwise really good job.

**Comment by aivyr** [comment]
@MJS103 thanks! changed the lines as per your suggestions.

**Comment by MJS103** [comment]
@aivyr If you want, can you change the reactions? I think it adds to the conversation, but you don't have to if you don't want to.

**Comment by aivyr** [comment]
the reactions?

**Comment by MJS103** [comment]
@aivyr The stuff before the quote -m [reaction] "quote"

**Comment by aivyr** [comment]
@MJS103 iirc if you don't specify an expression before a speech line, the expression from the last line is carried over? tested the code and that's how it works.

**Comment by monikLover** [comment]
Edited and reviewed. Edits and topics are approved. Good job on the writing tips!

**Pull request merged**